Sources/SwiftLexicalLookup/LookupName.swift

```diff
@@ -319,6 +338,20 @@ import SwiftSyntax
       return "implicit: \(strName)"
     case .dollarIdentifier(_, strRepresentation: let str):
       return "dollarIdentifier: \(str)"
+    case .compositeName(let names):
+      var result = "Composite name: [ "
+
+      for (index, name) in names.enumerated() {
+        result += name.debugDescription
+
+        if index < names.count - 1 {
+          result += ", "
+        } else {
+          result += " ]"
+        }
+      }
```

Comment:
Any reason why we can’t do 
```suggestion
      return "Composite name: [ \(names.map(\.debugDescription).joined(separator: ", "))]"
```

Reply:
Thanks for the suggestion! I replaced the old concatenation with this one.